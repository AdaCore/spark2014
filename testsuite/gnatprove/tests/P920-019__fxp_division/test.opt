ALL XFAIL division with Ada.Calendar.Time (P920-019)
