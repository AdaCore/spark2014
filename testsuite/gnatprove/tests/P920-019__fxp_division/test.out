delay_cal1.adb:7:68: medium: divide by zero might fail
delay_cal1.adb:7:68: medium: overflow check might fail
delay_cal1.adb:7:68: warning: universal_fixed expression interpreted as type "Standard.Duration"
